src/main.py

```diff
@@ -4,7 +4,7 @@
 
 from copa_modules import _types, data_processor
 
-BASE_PATH = "copa/"
+BASE_PATH = "TABLES_ADD_20240515/"
```

Comment:
Perhaps you should have multiple Python  files (not just main), so that you can execute different tasks at different times?

Reply:
Sure. I can do that.